go.mod

```diff
@@ -3,69 +3,68 @@ module github.com/kubernetes-csi/csi-lib-utils
 go 1.12
 
 require (
-	github.com/container-storage-interface/spec v1.1.0
+	github.com/container-storage-interface/spec v1.2.0
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
-	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
-	github.com/golang/protobuf v1.3.2
-	github.com/googleapis/gnostic v0.2.0 // indirect
-	github.com/onsi/ginkgo v1.10.2 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
-	github.com/stretchr/testify v1.4.0
-	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
-	golang.org/x/sys v0.0.0-20191220220014-0732a990476f // indirect
-	google.golang.org/genproto v0.0.0-20191220175831-5c49e3ecc1c1 // indirect
-	google.golang.org/grpc v1.26.0
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.1-beta.0 // indirect
-	k8s.io/client-go v0.17.0
-	k8s.io/component-base v0.17.0
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/protobuf v1.3.5
+	github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/onsi/ginkgo v1.12.0 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/stretchr/testify v1.5.1
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
+	golang.org/x/sys v0.0.0-20200406155108-e3b113bbe6a4 // indirect
+	google.golang.org/genproto v0.0.0-20200406120821-33397c535dc2 // indirect
+	google.golang.org/grpc v1.28.0
+	k8s.io/api v0.18.0
+	k8s.io/client-go v1.5.1
```

Comment:
Is this the correct version of client-go?

Reply:
ah.. corrected it @msau42